packages/atoms/src/SearchInput.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { TextInput, Pressable, View } from 'react-native';
+import { Box } from './Box';
+import { Icon } from './Icon/Icon';
+
+const MAX_LENGTH = 255;
+
+type InputProps = {
+    value: string;
+    onChange: (value: string) => void;
+    placeholder: string;
+    isClearable?: boolean;
+    isDisabled?: boolean;
+};
+
+type InputStyleProps = {
```

Comment:
please keep StyleProps right above style where it is used

Reply:
[8dde2c5](https://github.com/trezor/trezor-suite/pull/5433/commits/8dde2c5f1c1693cc984724aa9c04277bf7047375)